### ISSUE TITLE
Code fix for SA1009 removes trivia

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1187]
When the code fix for SA1009 is executed on the following call, the comment is removed.

```csharp
x.MethodName(
    arg1
    , _ => Assert.True(true) // comment
    , _ => Assert.True(true);
```

### Actual result

```csharp
testCallBack.AssertOnCallbackBeforeAndAfter(
    TimeOutMs
    , ev => Assert.True(true) , ev => Assert.AreEqual(isSuccess, ev.Success));
```

### Expected result

I'm actually not sure here. Perhaps no code fix offered? I'm open to suggestions.

**Comment by vweijsters** [comment]
I think no code fix should be offered. The comment should not be removed as it may contain important information. The location of the comment should also not be changed significantly as that could have an impact on the message in the comment as well.

In the example there is a trivial fix, even with keeping the comment, but that is not the general case.
It will be a lot of work to anticipate all possible tokens following the closing parenthesis and it may not always be possible to offer a proper fix that respects the observations above and that does not introduce another diagnostic.

As a side note, the actual result above would also produce SA1001, due to the spaces surrounding the comma, something that should be avoided as well.

**Comment by Noryoko** [comment]
SA1113 and SA1001 are reported when run through StyleCop. SA1009 is not reported.

**Comment by sharwell** [comment]
@Noryoko you are talking about StyleCop classic?

**Comment by Noryoko** [comment]
Yes.
